applications/audio/broadcast_configuration_tool/src/main.c

```diff
@@ -293,37 +272,35 @@ static void audio_send(void const *const data, size_t size, uint8_t num_ch)
 	prev_ret = ret;
 }
 
-static void broadcast_create(struct broadcast_source_big *broadcast_param,
-			     struct bt_bap_lc3_preset *preset)
+static void broadcast_create(uint8_t big_index)
 {
-	static enum bt_audio_location location[2] = {BT_AUDIO_LOCATION_FRONT_LEFT,
-						     BT_AUDIO_LOCATION_FRONT_RIGHT};
-	static struct subgroup_config subgroups[2];
-
-	for (size_t i = 0; i < ARRAY_SIZE(subgroups); i++) {
-		subgroups[i].group_lc3_preset = *preset;
-		subgroups[i].num_bises = 2;
-		subgroups[i].context = BT_AUDIO_CONTEXT_TYPE_MEDIA;
-		subgroups[i].location = location;
+	if (big_index >= CONFIG_BT_ISO_MAX_BIG) {
+		LOG_ERR("BIG index out of range");
+		return;
 	}
 
-	broadcast_param->packing = BT_ISO_PACKING_INTERLEAVED;
+	struct broadcast_source_big *brdcst_param = &broadcast_param[big_index];
 
-	broadcast_param->encryption = false;
+	for (size_t i = 0; i < ARRAY_SIZE(subgroups[big_index]); i++) {
+		if (subgroups[big_index][i].num_bises == 0) {
```

Comment:
Should we rather set `num_bises` to 2 at init and print a warning here if the user set it to 0?

Reply:
This is where the BIG is created, so I don't think we should set the number of BISes in a BIG that hasn't been created to 2. We could probably do a "clear data" type function if the num_bises == 0, since there is no "delete" function at the moment